evap/staff/forms.py

```diff
@@ -391,10 +391,11 @@ def clean(self):
 
 
 class ContributionFormSet(AtLeastOneFormSet):
-    def __init__(self, data=None, *args, **kwargs):
+    def __init__(self, data=None, can_edit_responsibles=True, *args, **kwargs):
```

Comment:
in the template this is called `can_change_responsible`, please make that consistent